### PR TITLE
Update runsettings file

### LINKED ISSUE #280: 🚧Update runsettings file
### I have done the items below . . .

- [X] I have updated the title without removing the 🚧 emoji.

### Description

Update the `Velaptor.runsettings` file by fixing one of the exclude items.

Currently it is excluding an old name of a library that does not exist anymore and it should be excluding the entire _**VelapatorTesting**_ app code base.

Need to run this in _**Visual Studio**_ to make sure that the runsettings is correctly ignoring the **VelaptorTesting** code base.

**Refer to image below:**
image

### Acceptance Criteria

**This issue is finished when:**
- [x] The value `VelaptorTesting.dll` added to exclusions
- [x] The value `VelaptorTesting.exe` added to exclusions
- [x] All unit tests pass

### ToDo Items

- [X] Priority label added to this issue.  Refer to the _**Priority Type Labels**_ section below.
- [X] Change type labels added to this issue.  Refer to the _**Change Type Labels**_ section below.
- [X] Issue linked to the correct project.
- [X] Issue linked to the correct milestone.
- [x] Draft pull request created and linked to this issue.

### Issue Dependencies

_No response_

### Related Work

_No response_

### Additional Information:

**_<details closed><summary>Change Type Labels</summary>_**

| Change Type         | Label                                                                                |
|---------------------|--------------------------------------------------------------------------------------|
| Bug Fixes           | https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%90%9Bbug                     |
| Breaking Changes    | https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%92%A3breaking%20changes      |
| Enhancement         | https://github.com/KinsonDigital/Velaptor/labels/enhancement                         |
| Workflow Changes    | https://github.com/KinsonDigital/Velaptor/labels/workflow                            |
| Code Doc Changes    | https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%93%91documentation%2Fcode    |
| Product Doc Changes | https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%93%9Ddocumentation%2Fproduct |
</details>

**_<details closed><summary>Priority Type Labels</summary>_**

| Priority Type       | Label                                                              |
|---------------------|--------------------------------------------------------------------|
| Low Priority        | https://github.com/KinsonDigital/Velaptor/labels/low%20priority    |
| Medium Priority     | https://github.com/KinsonDigital/Velaptor/labels/medium%20priority |
| High Priority       | https://github.com/KinsonDigital/Velaptor/labels/high%20priority   |
</details>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [x] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the projects [branching documentation](../../Documentation/Branching.md).

- [x] Pull request title matches the title of the associated issue.

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out and let's discuss it.

- [x] A preview-label label has been added to the pull request.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - This does not mean that the tests pass in both the local and CI environment.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

**Review by KinsonDigitalAdmin — DISMISSED** [review]

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.9.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ As with all software, including releases, there is always a chance for issues an
 
 <h2 style="font-weight:bold" align="center">Nuget/Library Updates 📦</h2>
 
-1. [#209](https://github.com/KinsonDigital/Velaptor/issues/209) - Updated nuget package **Microsoft.CodeAnalysis.NetAnalyzers** from **_5.0.3_** to **_v6.0.0_**
+1. [#209](https://github.com/KinsonDigital/Velaptor/issues/209) - Updated nuget package **Microsoft.CodeAnalysis.NetAnalyzers** from **v_5.0.3_** to **_v6.0.0_**
```
Comment (KinsonDigitalAdmin) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.9.md:75]: Refactor to example below

```suggestion
1. [#209](https://github.com/KinsonDigital/Velaptor/issues/209) - Updated nuget package **Microsoft.CodeAnalysis.NetAnalyzers** from **_v5.0.3_** to **_v6.0.0_**
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

**Review by KinsonDigitalAdmin — DISMISSED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]
